data_structures/stack.py

```diff
@@ -31,6 +31,9 @@ def main():
     s.push(5)
     print(s.pop())
     print(s.peek())
+    print(s.pop())
```

Comment:
@nealav Were you able to review this PR? :)

Reply:
I can't accept any PRs without proper unit tests